scripts/lv_conf_checker.py

```diff
@@ -27,13 +27,16 @@
 #define LV_CONF_INTERNAL_H
 /* clang-format off */
 
-/*Handle special Kconfig options*/
-#include "lv_conf_kconfig.h"
-
 #include <stdint.h>
 
+#if defined (ESP_PLATFORM)
+#include "sdkconfig.h"
+#include "esp_attr.h"
+#include "lv_conf_kconfig.h"
```

Comment:
Shouldn't `lv_conf_kconfig.h` be included in all cases? ESP-IDF is not the only framework we are targeting for this.

Reply:
You're right, I was just being safe to don't break anything else.

Reply:
Fixed with c3698d6